src/form/Checkbox/CheckboxLabel.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { FC } from 'react';
+import { twMerge } from 'tailwind-merge';
+import { CheckboxTheme } from './CheckboxTheme';
+
+interface CheckboxLabelProps {
+  label: string;
+  size: 'small' | 'medium' | 'large' | string;
+  disabled?: boolean;
+  checked?: boolean;
+  onChange?: () => void;
+  labelClassName?: string;
+  theme: CheckboxTheme;
+}
+
+export const CheckboxLabel: FC<CheckboxLabelProps> = ({
+  label,
+  size,
+  disabled,
+  checked,
+  onChange,
+  labelClassName,
+  theme
+}) => {
+  return (
+    <span
```

Comment:
can remove the unnecessary return. Im gonna add an eslint rule for this real quick 😁

Reply:
done